tests/dicts/test_benedict_wildcard.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from benedict import benedict
+
+
+class keypath_dict_list_wildcard_test_case(unittest.TestCase):
+    def setUp(self):
+        self.d = {
```

Comment:
Declare the data that each test method needs at the beginning of the method, each test method should be independent/isolated if possible, there is also less risk to compromise tests result when changing/adjusting the data for a specific method..

Reply:
I removed it for consistency. But this method is run before each [test method](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp).